basic/okta_sdk/impl.py

```diff
@@ -1,17 +1,43 @@
 from sym.sdk.annotations import hook, reducer
 from sym.sdk.integrations import okta, slack
 from sym.sdk.templates import ApprovalTemplate
+from sym.sdk.notifications import Notification
+from sym.sdk.request_permission import PermissionLevel, RequestPermission
 
 
 # Reducers fill in the blanks that your workflow needs in order to run.
 @reducer
-def get_approvers(event):
-    """Route Sym requests to a channel based on the user's Okta profile."""
+def get_permissions(event):
+    """Decide who can see and take actions on requests."""
+
+    department = get_user_department(event.user)
+    approve_deny = slack.users_in_channel("#sym-requests")
+    if department == "engineering":
+        approve_deny = slack.users_in_channel("#eng-requests")
```

Comment:
Would it make more sense for these to pull groups of Okta users? Something like:

```python
department = get_user_department(event.user)
engineering_managers_okta_group_id = "00g12345"
other_managers_okta_group_id = "00g9876"

if department == "engineering":
    approve_deny = user_ids(okta.users_in_group(group_id=engineering_managers_okta_group_id))
else:
    approve_deny = user_ids(okta.users_in_group(group_id=other_managers_okta_group_id))
```

Would want to know if @sym-andrewa has thoughts too

Reply:
Yeah, agree that using `okta.users_in_group()` makes more sense than `slack.users_in_channel()`, especially given that this is an Okta SDK example. (Also, either way, you need to wrap in `user_ids()` to use with `RequestPermissions`.)